catroid/src/org/catrobat/catroid/ui/fragment/LookFragment.java

```diff
@@ -780,6 +788,34 @@ protected void showDeleteDialog() {
 		deleteLookDialog.show(getFragmentManager(), DeleteLookDialog.DIALOG_FRAGMENT_TAG);
 	}
 
+	private void moveLookDataDown() {
+		if (selectedLookPosition < lookDataList.size() - 1) {
+			LookData down_element = lookDataList.get(selectedLookPosition + 1);
```

Comment:
Checkstyle also covers snake case
@bernadettespieler: ...so please take a look at the jenkins result.